tests/e2e/default/more-options-menu/online-user/permissions-enabled.wdio-spec.js

```diff
@@ -106,12 +106,12 @@ describe('Online User', async () => {
       await sms.sendSms('testing', contact.phone);
     });
 
-    it('- Reports tab - Edit/export invisible when NON XML report selected', async () => {
+    it('- Reports tab - Edit invisible when NON XML report selected', async () => {
       await commonPage.goToReports();
       await reportPage.goToReportById(smsReportId);
-      await reportPage.reportBodyDetails().waitForDisplayed();
+      await (await reportPage.reportBodyDetails()).waitForDisplayed();
       await commonPage.openMoreOptionsMenu();
-      expect(await commonPage.isMenuOptionVisible('export', 'reports')).to.be.false;
+      expect(await commonPage.isMenuOptionVisible('export', 'reports')).to.be.true;
```

Comment:
the condition to `not` display export button is: 
```
 !this.selectMode && this.isOnlineOnly && this.hasExportPermission && !this.responsiveService.isMobile();
```

https://github.com/medic/cht-core/blob/07bcd95e2858f12b339777da5bcb05c63e470c8e/webapp/src/ts/modules/reports/reports-more-menu.component.ts#L142

Nothing in this test would prevent the export option from being displayed. I'm surprised this test ever passed.

Reply:
It has been flaky though... https://github.com/medic/cht-core/issues/8091

Reply:
Makes you wonder about all of our other passing tests :D

Reply:
@tatilepizs this is what I was telling you about.

Reply:
🤯 
Same happened with the test `add-user test.wdio-spec.js` that I fixed in [this PR](https://github.com/medic/cht-core/pull/8670), I don't understand how it was passing